com.automattic.beeper.yml

```
@@ -0,0 +1,65 @@
+app-id: com.automattic.beeper
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: start-beeper
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-documents
+  - --device=dri
+#  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.Unity
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --env=ELECTRON_TRASH=gio
+  - --env=ELECTRON_OZONE_PLATFORM_HINT=auto
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: beeper
+    buildsystem: simple
+    cleanup:
+      - squashfs-root
+    sources:
+      - type: file
+        url: https://beeper-desktop.download.beeper.com/builds/Beeper-4.0.478.AppImage
+        sha256: 49a60e62b4afce1602f2e3cf11b04c4d3663381483c60e22537a807f965a413b
+        dest-filename: beeper.AppImage
+
+      - type: script
+        dest-filename: start-beeper.sh
+        commands:
+          - exec zypak-wrapper /app/beepertexts --no-sandbox "$@"
```

```suggestion
          - exec zypak-wrapper /app/beepertexts "$@"
```

you don't need no-sandbox with zypak.